ospfd/ospf_te.c

```diff
@@ -2563,7 +2563,8 @@ DEFUN (show_ip_ospf_mpls_te_link,
 		FOR_ALL_INTERFACES (vrf, ifp)
 			show_mpls_te_link_sub(vty, ifp);
 		return CMD_SUCCESS;
-	}
+	} else
+		ospf = ospf_lookup_by_vrf_id(VRF_DEFAULT);
```

Comment:
the `} else` is not necessary due to the `return CM_SUCCESS` at the end of the if.
and `ospf = ospf_lookup_by_vrf_id(VRF_DEFAULT);` will be better place at line 2579 as ospf variable is modified in the `if` block lines 2569 - 2577